### PR TITLE
Add missing user tests

### PR DESCRIPTION
**Pull request by kookster**

### CODE REVIEW

File: test/models/address_test.rb [hunk 1]
```diff
@@ -2,9 +2,19 @@
 
 describe Address do
 
-  let(:address) { FactoryGirl.create(:address) }
+  let(:address) { create(:address) }
```
Comment (chrisrhoden) [inline test/models/address_test.rb]: this is not exercised

Comment (kookster) [inline test/models/address_test.rb]: true, I will remove it.

File: test/models/address_test.rb [hunk 2]
```diff
@@ -2,9 +2,19 @@
...
 
   it 'has a table defined' do
     Address.table_name.must_equal 'addresses'
   end
+
+  it 'provides underlying account for user address' do
+    user.address.account.must_equal user.individual_account
+  end
+
+  it 'provides underlying account for user address' do
```
Comment (kookster) [inline test/models/address_test.rb]: this is also a dupe and wrong desc, will update it.

**Comment by kookster** [comment]
@chrisrhoden lgtm now.

**Pull request merged**